MMM-Wunderlist.js

```diff
@@ -47,44 +60,71 @@ Module.register("MMM-Wunderlist", {
         for (var todo in list) {
           if (this.config.order == "reversed") {
             tasksShown.push(list[todo]);
-          } else {
+          }
+          else {
             tasksShown.unshift(list[todo]);
           }
         }
       }
     }
     return tasksShown.slice(0, this.config.maximumEntries);
+  },
 
+  getScripts: function () {
+    return [
+      'String.format.js'
+    ];
   },
+  getStyles: function () {
+    return [
+      'https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css'
```

Comment:
Font-Awesome comes bundled with MM. Please use the local version! 
https://github.com/MichMich/MagicMirror/blob/master/vendor/vendor.js